user_sync/rules.py

```diff
@@ -746,18 +751,18 @@ def is_umapi_user_excluded(self, in_primary_org, user_key, current_groups, do_lo
             identity_type, username, domain = self.parse_user_key(user_key)
             if identity_type in self.exclude_identity_types:
                 if do_logging:
-                    self.logger.info("Excluding adobe user (due to type): %s", user_key)
+                    self.logger.debug("Excluding adobe user (due to type): %s", user_key)
                 self.excluded_user_count += 1
                 return True
             if len(current_groups & self.exclude_groups) > 0:
                 if do_logging:
-                    self.logger.info("Excluding adobe user (due to group): %s", user_key)
+                    self.logger.debug("Excluding adobe user (due to group): %s", user_key)
                 self.excluded_user_count += 1
                 return True
             for re in self.exclude_users:
                 if re.match(username):
                     if do_logging:
-                        self.logger.info("Excluding adobe user (due to name): %s", user_key)
+                        self.logger.debug("Excluding adobe user (due to name): %s", user_key)
                     self.excluded_user_count += 1
```

Comment:
I slightly ambivalent abuot these 3 but let's change it and see if anyone complains.